lib/os_places_api/location_builder.rb

```diff
@@ -4,12 +4,11 @@ def initialize(result)
       @result = result
     end
 
-    def build_location
-      Location.new(postcode: @result.dig("DPA", "POSTCODE"),
-                   address: @result.dig("DPA", "ADDRESS"),
-                   latitude: @result.dig("DPA", "LAT"),
-                   longitude: @result.dig("DPA", "LNG"),
-                   local_custodian_code: @result.dig("DPA", "LOCAL_CUSTODIAN_CODE"))
+    def build_location(dataset)
```

Comment:
No need to add this parameter - looks like this slipped in from the next commit 👀 

Also this first commit has a typo ("form" instead of "from")